lib/Runtime/Library/JSON.cpp

```diff
@@ -706,25 +706,26 @@ namespace JSON
 
         if(isEmpty)
         {
-            result = scriptContext->GetLibrary()->CreateStringFromCppLiteral(_u("{}"));
+            result = scriptContext->GetPropertyString(PropertyIds::object_Empty);
```

Comment:
No, we may cache it under library. That's why didn't touch the byte codes. I'm trying something else (on use caching for library) and see how it affects the perf. So many by default cached members..